meson.build

```diff
@@ -37,19 +37,27 @@ internal_conf = configuration_data()
 mlibc_conf = configuration_data()
 
 if not headers_only
-	cxxshim_dep = subproject('cxxshim').get_variable('cxxshim_dep')
+	cxxshim_dep = dependency('cxxshim', method: 'pkg-config', fallback: ['cxxshim', 'cxxshim_dep'])
 	libc_deps += cxxshim_dep
 	rtdl_deps += cxxshim_dep
-	frigg_dep = subproject('frigg',
-		default_options: ['frigg_no_install=true']).get_variable('frigg_dep')
+
+	# Ideally we'd use the same one-liner as above, but we can't pass default_options that way
```

Comment:
Why not? The `dependency()` function absolutely does document that you can pass default_options to it and those options will be applied in the case that subproject fallback is done.

Reply:
Oh, you're absolutely right, I just missed that in the docs

Reply:
Thanks :)